clusterloader2/api/validation.go

```diff
@@ -105,6 +108,35 @@ func (v *ConfigValidator) validateStep(s *Step, fldPath *field.Path) field.Error
 	return allErrs
 }
 
+func (v *ConfigValidator) handleTiningSetInPhase() field.ErrorList {
+	allErrs := field.ErrorList{}
+	c := v.config
+	for i := range c.Steps {
+		allErrs = append(allErrs, validateTuningSetInPhase(c.Steps[i], c.TuningSets, field.NewPath("phase").Index(i))...)
```

Comment:
`field.NewPath("phase").Index(i))` is reason why you see `"phase[0]"` in the test

Reply:
Thanks, fixed!